NitroxServer/ConsoleCommands/OpCommand.cs

```diff
@@ -6,18 +6,17 @@ namespace NitroxServer.ConsoleCommands
 {
     internal class OpCommand : Command
     {
-        public OpCommand() : base("op", Perms.ADMIN, "Sets an user as admin")
+        public OpCommand() : base("op", Perms.ADMIN, "Sets a user as admin")
```

Comment:
```suggestion
        public OpCommand() : base("op", Perms.ADMIN, "Sets an user as admin")
```

Reply:
`a` is correct here https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

Reply:
Ok sry :D

Reply:
I discovered that weird rule few days ago too